src/app/service/guard/login.guard.ts

```diff
@@ -0,0 +1,14 @@
+import { inject } from '@angular/core'
+import { CanActivateFn, Router } from '@angular/router'
+import { UsersService } from '../users/users.service'
+
+export const loginGuard: CanActivateFn = (route, state) => {
+  const authService = inject(UsersService)
+  const router = inject(Router)
+
+  if (authService.isUserSignedIn()) {
+    router.navigate(['/tasks-list'])
+    return true
```

Comment:
you can return true at the end, no need to return true inside the if statement if in the end it will be true anyways